Formula/vsh.rb

```diff
@@ -0,0 +1,29 @@
+class Vsh < Formula
+  desc "HashiCorp Vault interactive shell"
+  homepage "https://github.com/fishi0x01/vsh"
+  url "https://github.com/fishi0x01/vsh/archive/v0.7.2.tar.gz"
+  sha256 "f20885425a9c6ee04ed447046e6df113585469b37332e02b7a23639b718314a1"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "vault" => :test
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-X main.vshVersion=v#{version}"
+  end
+
+  test do
+    require "expect"
+    version_output = shell_output("#{bin}/vsh -version")
+    assert_match version.to_s, version_output
+    error_output = shell_output("#{bin}/vsh -c \"ls\" 2>&1", 1)
+    assert_match "Error initializing vault client | Is VAULT_ADDR properly set?", error_output
```

Comment:
TBH I think you can stop here and this test would be sufficient. I also suspect you can drop the `\"` around `ls` above.

Reply:
Ok. Commented out the rest, so if vault gets bottled it can be used. Or should I remove it completely?

Reply:
Hmm. Not sure. Let's wait and see if someone else will chime in with an opinion.

Reply:
I think we could do without the actual vault based check. I'd just remove it.

Reply:
Alright, I removed the comments.